src/app/utils/rpc-utils.ts

```diff
@@ -13,6 +13,7 @@ const RPC_ENDPOINTS: Record<Network, Partial<Record<Layer, string>>> = {
   [Network.testnet]: {
     [Layer.emerald]: 'https://testnet.emerald.oasis.dev',
     [Layer.sapphire]: 'https://testnet.sapphire.oasis.dev',
+    [Layer.pontusx]: 'https://grpc.sapphire.testnet.oasiscloud.io:443',
```

Comment:
notes from discussions:
- this isn't publicly accessible, but backend has access. In the future nexus will get the balance from RPC
- Frontend can get the balances from https://testnet.grpc.oasis.dev if we replace [our web3 RPC fetching](https://github.com/oasisprotocol/explorer/blob/7dc2a0e0f3673f6b022e69a34ef04497d28be3a0/src/app/utils/rpc-utils.ts#L1) with oasis RPC
```js
// node node-get-runtime-balance.mjs
import oasis from '@oasisprotocol/client'
import oasisRT from '@oasisprotocol/client-rt'
import xhr2 from 'xhr2'
global.XMLHttpRequest = xhr2 // fixes oasis sdk in nodejs

  const testnetNic = new oasis.client.NodeInternal('https://testnet.grpc.oasis.dev')
  const accountsWrapper = new oasisRT.accounts.Wrapper(oasis.misc.fromHex('0000000000000000000000000000000000000000000000004febe52eb412b421'));

  const daveEthAddr = '0x9c26685b6E8e2997d9aAf3f1a642f1b1b3dB9580';
  const daveEthAddrU8 = oasis.misc.fromHex(daveEthAddr.slice(2));
  const daveAddr = await oasis.address.fromData(
    oasisRT.address.V0_SECP256K1ETH_CONTEXT_IDENTIFIER,
    oasisRT.address.V0_SECP256K1ETH_CONTEXT_VERSION,
    daveEthAddrU8,
  );

  (await accountsWrapper.queryBalances().setArgs({address: daveAddr,}).query(testnetNic)).balances
    .forEach((balance, name) => console.log(oasis.misc.toStringUTF8(name) || 'native', oasis.quantity.toBigInt(balance)));
  // native 908027345930000000000n
  // TEST 10000000000000000000n

  // https://explorer.dev.oasis.io/testnet/pontusx/address/0x9c26685b6E8e2997d9aAf3f1a642f1b1b3dB9580
  // 908.02734593 EUROe
  // 10 TEST
```